### PR TITLE
Tag Gillespie v0.0.2 [https://github.com/sdwfrost/Gillespie.jl]

### PR DESCRIPTION
**Pull request by sdwfrost**

### CODE REVIEW

**Comment by ararslan** [comment]
Looks like you don't have a Julia version specified in the requires file, which is why Travis is failing.

File: Gillespie/versions/0.0.2/sha1 [hunk 1]
```diff
@@ -0,0 +1 @@
+dc8e829603a62c953033d796b2b74709a25ade0d
```
Comment (tkelman) [inline Gillespie/versions/0.0.2/sha1]: are you sure this is the commit you want to tag? it's from July https://github.com/sdwfrost/Gillespie.jl/commit/dc8e829603a62c953033d796b2b74709a25ade0d

File: Gillespie/versions/0.0.2/requires [hunk 2]
```diff
@@ -0,0 +1,5 @@
+julia 0.4
+Compat
```
Comment (tkelman) [inline Gillespie/versions/0.0.2/requires]: should declare a minimum version dependency that implements the features you need

File: Gillespie/versions/0.0.2/sha1 (hunk 1 above)
Comment (sdwfrost) [inline Gillespie/versions/0.0.2/sha1]: My bad; I didn't update the package from the Git repo before publishing, as I started with a fresh install. I actually want to tag this commit: [https://github.com/sdwfrost/Gillespie.jl/commit/39fb53516cc1d736c18ba7ac62d51e81287f9246](https://github.com/sdwfrost/Gillespie.jl/commit/39fb53516cc1d736c18ba7ac62d51e81287f9246)

**Pull request closed without merging**